```
wget https://www.openssl.org/source/openssl-1.0.1t.tar.gz
--2017-05-25 16:20:11--  https://www.openssl.org/source/openssl-1.0.1t.tar.gz
Resolving www.openssl.org (www.openssl.org)... 104.126.148.122, 2600:1405:6:180::c1e, 2600:1405:6:18b::c1e
Connecting to www.openssl.org (www.openssl.org)|104.126.148.122|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-05-25 16:20:12 ERROR 404: Not Found.

make: *** [openssl-1.0.1t/Configure] Error 8
program finished with exit code 2
elapsedTime=2.563634
```

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/context/context-attributes-alpha-depth-stencil-antialias.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/context/context-attributes-alpha-depth-stencil-antialias.html.ini
@@ -1,11 +1,5 @@
 [context-attributes-alpha-depth-stencil-antialias.html]
   type: testharness
-  expected:
-    if os == "mac": CRASH
-    if os == "linux": TIMEOUT
-  [WebGL test #0: gl = getWebGL(1, 1, null, [ 0, 0, 0, 0 \], 1, 0) should be non-null. Was null]
-    expected: FAIL
-
-  [WebGL test #0: gl = getWebGL(1, 1, null, [ 0, 0, 0, 0 \], 1, 0) should be non-null. Threw exception TypeError: argument could not be converted to any of: ImageData, HTMLImageElement, HTMLCanvasElement, HTMLVideoElement]
+  [WebGL test #95: pixel_1[0\] != 255 && pixel_1[0\] != 0 || pixel_1[0\] == 0 && pixel_2[0\] != 255 && pixel_2[0\] != 0 should be true. Was false.]
     expected: FAIL
 

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/extensions/oes-texture-float-with-canvas.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/extensions/oes-texture-float-with-canvas.html.ini
@@ -3,3 +3,4 @@
   expected: ERROR
   [Overall test]
     expected: NOTRUN
+

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/extensions/oes-texture-half-float-with-canvas.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/extensions/oes-texture-half-float-with-canvas.html.ini
@@ -3,3 +3,4 @@
   expected: ERROR
   [Overall test]
     expected: NOTRUN
+

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/rendering/gl-scissor-test.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/rendering/gl-scissor-test.html.ini
@@ -51,9 +51,54 @@
   [WebGL test #97: getError expected: NO_ERROR. Was INVALID_ENUM : there should be no errors]
     expected: FAIL
 
-  [WebGL test #98: Unable to fetch WebGL rendering context for Canvas]
+  [WebGL test #148: at (0, 0) expected: 0,255,0,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #99: context does not exist]
+  [WebGL test #151: at (1, 1) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #154: at (2, 2) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #157: at (3, 3) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #160: at (4, 4) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #163: at (5, 5) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #166: at (6, 6) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #169: at (7, 7) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #172: at (8, 8) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #175: at (9, 9) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #178: at (10, 10) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #181: at (11, 11) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #184: at (12, 12) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #187: at (13, 13) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #190: at (14, 14) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #193: at (15, 15) expected: 0,255,0,255 was 0,0,0,0]
+    expected: FAIL
+
+  [WebGL test #195: getError expected: NO_ERROR. Was INVALID_ENUM : there should be no errors]
     expected: FAIL
 

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/rendering/gl-viewport-test.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/rendering/gl-viewport-test.html.ini
@@ -39,156 +39,42 @@
   [WebGL test #81: getError expected: NO_ERROR. Was INVALID_ENUM : there should be no errors]
     expected: FAIL
 
-  [WebGL test #82: Unable to fetch WebGL rendering context for Canvas]
+  [WebGL test #83: at (16, 32) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #83: context does not exist]
+  [WebGL test #88: at (8, 8) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #2: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #93: at (4, 16) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #3: at (48, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #98: at (32, 64) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #4: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #123: at (32, 96) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #5: at (0, 96) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #128: at (24, 72) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #7: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #133: at (36, 48) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #8: at (24, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #138: at (48, 96) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #9: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #143: at (0, 32) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #10: at (0, 24) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #148: at (8, 0) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #12: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #153: at (0, 16) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #13: at (12, 0) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #158: at (32, 0) expected: 0,0,255,255 was 0,0,0,0]
     expected: FAIL
 
-  [WebGL test #14: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #15: at (0, 48) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #17: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #19: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #23: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #24: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #25: at (0, 96) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #27: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #28: at (48, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #30: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #32: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #34: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #35: at (0, 96) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #37: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #38: at (48, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #39: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #42: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #44: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #47: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #48: at (40, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #49: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #50: at (0, 88) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #52: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #53: at (44, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #54: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #55: at (0, 80) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #57: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #59: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #63: at (32, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #64: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #65: at (0, 96) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #67: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #68: at (24, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #70: at (0, 64) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #73: at (32, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #74: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #75: at (0, 48) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #77: at (0, 0) expected: 0,0,0,0 was 153,153,153,255]
-    expected: FAIL
-
-  [WebGL test #80: at (0, 64) expected: 0,0,0,0 was 153,153,153,255]
+  [WebGL test #163: getError expected: NO_ERROR. Was INVALID_ENUM : there should be no errors]
     expected: FAIL
 

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/textures/copy-tex-image-and-sub-image-2d.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/textures/copy-tex-image-and-sub-image-2d.html.ini
@@ -1,9 +1,578 @@
 [copy-tex-image-and-sub-image-2d.html]
   type: testharness
-  expected: ERROR
-  [WebGL test #0: Unable to fetch WebGL rendering context for Canvas]
+  [WebGL test #85: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
     expected: FAIL
 
-  [Overall test]
-    expected: NOTRUN
+  [WebGL test #90: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #94: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #95: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #99: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #100: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #104: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #105: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #109: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #110: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #114: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #119: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #133: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #135: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #138: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #140: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #142: at (0, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #143: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #144: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #145: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #147: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #148: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #149: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #150: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #152: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #153: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #154: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #155: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #157: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #158: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #159: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #160: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #162: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #164: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #165: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #167: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #169: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #170: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #174: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #175: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #179: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #180: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #183: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #184: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #185: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #188: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #189: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #190: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #192: at (0, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #193: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #194: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #195: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #197: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #198: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #199: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #200: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #202: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #203: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #204: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #205: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #207: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #208: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #209: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #210: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #212: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #213: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #214: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #215: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #217: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #218: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #219: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #220: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #222: at (0, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #223: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #224: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #225: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #227: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #228: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #229: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #230: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #232: at (0, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #233: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #234: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #235: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #237: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #238: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #239: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #240: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #242: at (0, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #243: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #244: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #245: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #247: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #248: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #249: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #250: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #336: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #341: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #345: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #346: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #350: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #351: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #355: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #356: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #360: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #361: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #365: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #370: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #384: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #386: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #389: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #391: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #393: at (0, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #394: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #395: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #396: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #398: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #399: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #400: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #401: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #403: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #404: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #405: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #406: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #408: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #409: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #410: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #411: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #413: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #415: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #416: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #418: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #420: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #421: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #425: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #426: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #430: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #431: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #434: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #435: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #436: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #439: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #440: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #441: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #443: at (0, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #444: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #445: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #446: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #448: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #449: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #450: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #451: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #453: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #454: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #455: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #456: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #458: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #459: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #460: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #461: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #463: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #464: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #465: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #466: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #468: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #469: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #470: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #471: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #473: at (0, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #474: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #475: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #476: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #478: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #479: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #480: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #481: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #483: at (0, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #484: at (1, 0) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #485: at (0, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #486: at (1, 1) expected: 0,0,0,0 was 0,0,255,255]
+    expected: FAIL
+
+  [WebGL test #488: at (0, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #489: at (1, 0) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #490: at (0, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #491: at (1, 1) expected: 0,0,0,0 was 128,128,128,128]
+    expected: FAIL
+
+  [WebGL test #493: at (0, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #494: at (1, 0) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #495: at (0, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #496: at (1, 1) expected: 0,0,0,0 was 255,0,0,255]
+    expected: FAIL
+
+  [WebGL test #498: at (0, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #499: at (1, 0) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #500: at (0, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
+
+  [WebGL test #501: at (1, 1) expected: 0,0,0,0 was 0,255,0,255]
+    expected: FAIL
 

--- a/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/textures/tex-image-with-format-and-type.html.ini
+++ b/tests/wpt/metadata/webgl/conformance-1.0.3/conformance/textures/tex-image-with-format-and-type.html.ini
@@ -1,6 +1,5 @@
 [tex-image-with-format-and-type.html]
   type: testharness
-
   [WebGL test #6: LUMINANCE/UNSIGNED_BYTE should maintain full precision of data]
     expected: FAIL
 
@@ -36,3 +35,4 @@
 
   [WebGL test #70: UNPACK_PREMULTIPLY_ALPHA_WEBGL with RGBA/UNSIGNED_SHORT_5_5_5_1]
     expected: FAIL
+

--- a/tests/wpt/mozilla/meta/MANIFEST.json
+++ b/tests/wpt/mozilla/meta/MANIFEST.json
@@ -31658,7 +31658,7 @@
    "support"
   ],
   "mozilla/webgl/context_creation_error.html": [
-   "568b017c2db703c4976a8fa6757721f257a13d97",
+   "d6ffc0c4ea5671399d3c9b6440608b47c80699cf",
    "testharness"
   ],
   "mozilla/webgl/draw_arrays_simple.html": [

--- a/tests/wpt/mozilla/meta/mozilla/webgl/context_creation_error.html.ini
+++ b/tests/wpt/mozilla/meta/mozilla/webgl/context_creation_error.html.ini
@@ -1,0 +1,4 @@
+[context_creation_error.html]
+  type: reftest
+  [WebGLContextEvent "webglcontextcreationerror" event]
+    expected: FAIL

--- a/tests/wpt/mozilla/tests/mozilla/webgl/context_creation_error.html
+++ b/tests/wpt/mozilla/tests/mozilla/webgl/context_creation_error.html
@@ -14,7 +14,8 @@ async_test(function() {
                 "'statusMessage' should be a string, " + typeof(e.statusMessage) + " found");
   }), false);
 
-  // Antialising actually is not supported and thus triggers the error event
+  // TODO: Create a dummy function to fail the webgl context forcefully from js tests.
+  // Now that antialias doesn't throw an error, there isn't a way to force context creation errors.
   var gl = canvas.getContext('webgl', { antialiasing: true });
 
   assert_false(!!gl, "WebGLContext creation succeeded, please update this test!");

